docs/developers/sort_inference.md

```diff
@@ -50,8 +50,8 @@ Informally then, the polarity restriction enforces that type intersections can o
 - `𝜏 <: 𝜏1 and 𝜏 <: 𝜏2 iff 𝜏 <: 𝜏1 ⊓ 𝜏2`
 - `𝜏1 <: 𝜏 and 𝜏2 <: 𝜏 iff 𝜏1 ⊔ 𝜏2 <: 𝜏`
 
-In total then, any type variable with bounds `L <: 𝛼 <: U` can be encoded as a set-theoretic type by 
-- replacing every negative instance of `𝛼` with `𝛼 ⊓ U` 
+In total then, any type variable with bounds `L <: 𝛼 <: U` can be encoded as a set-theoretic type by
+- replacing every negative instance of `𝛼` with `𝛼 ⊓ U`
```

Comment:
We probably don't want to enforce trailing whitespace in Markdown since it has a semantic meaning:
https://www.markdownguide.org/basic-syntax/#line-break-best-practices

Reply:
Good catch - looks like tabs are also significant (https://www.markdownguide.org/hacks/#indent-tab), so I'll remove the tabs-to-spaces as well